phpunit/blocks/render-last-posts-test.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Last post block rendering tests.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ */
+
+/**
+ * Tests for the Last post block.
+ *
+ * @group blocks
+ */
+class Tests_Blocks_RenderLastPosts extends WP_UnitTestCase {
+	protected static $post;
+	protected static $sticky_post;
+	protected static $attachment_id;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$sticky_post = $factory->post->create_and_get(
+			array(
+				'post_title' => 'Sticky post',
+				'post_date'  => '2008-09-03 00:00:00',
+			)
+		);
+		$factory->post->create_many( 2 );
+		self::$post = $factory->post->create_and_get();
+
+		stick_post( self::$sticky_post->ID );
+
+		$file                = DIR_TESTDATA . '/images/canola.jpg';
+		self::$attachment_id = $factory->attachment->create_upload_object( $file, self::$post->ID );
+		set_post_thumbnail( self::$post, self::$attachment_id );
+	}
+
+	public static function tear_down_after_class() {
+		wp_delete_post( self::$attachment_id, true );
+		parent::tear_down_after_class();
+	}
```

Comment:
```suggestion
	public static function wpTearDownAfterClass() {
		wp_delete_post( self::$attachment_id, true );
	}
```

Possibly a comment this is deleted in the tear down as you need to delete the file from the uploads directory, which the base class tear down doesn't do.